packages/@aws-cdk-containers/ecs-service-extensions/README.md

```diff
@@ -392,11 +392,42 @@ For setting up a topic-specific queue subscription, you can provide a custom que
 
 ```ts
 nameDescription.add(new QueueExtension({
-  queue: myEventsQueue,
+  eventsQueue: myEventsQueue,
   subscriptions: [new TopicSubscription({
     topic: new sns.Topic(stack, 'my-topic'),
     // `myTopicQueue` will subscribe to the `my-topic` instead of `eventsQueue`
-    queue: myTopicQueue,
```

Comment:
follow-up: if it is a breaking change, I don't think we can allow that since this is a stable module. Although I am not sure if there is some exception for this module, since it is separate from the @aws-cdk namespace.

Reply:
Good catch! It is a breaking change and we follow the same policy for breaking changes as the @aws-cdk. I have deprecated the `queue` property of the `TopicSubscription` class.